pages/docs/features/realtime.mdx

```diff
@@ -77,8 +99,25 @@ export const inngest = new Inngest({
 
 Now, in your Inngest functions, the `publish()` function will be available as a parameter to your handler function. When publishing data, you'll need to specify the `channel` and `topic` you want to publish to and any data you want to publish.
 
-<CodeGroup>
+</GuideSection>
+
+<GuideSection show="python">
+
+```py {{ title: "client.py" }}
+import logging
+
+import inngest
 
+logger = logging.getLogger("uvicorn.inngest")
+logger.setLevel(logging.DEBUG)
+
+inngest_client = inngest.Inngest(app_id="fast_api_example", logger=logger)
+```
```

Comment:
I don't think we need to cover the Python client here as that should be a pre-requesite. The only reason we show the client in the TS SDK example is b/c it requires middleware whereas Python has realtime built in, just needs to import.

Reply:
Reading onward, I see why you did this, b/c the client is used later. Perhaps we can instead link to another doc that says "You'll need to have set tup the client which you would have learned in our Python quick start guide" - or something like that.

Reply:
Yeah, I think this makes sense. I think I'll role the subscribe doc back into this and then next phase of realtime docs will add more explanation around subscribing and so on so I can go into detail on sdk's and subscribing, clients and so on.